### PR TITLE
enable illustrative color scheme in the tree

### PR DESCRIPTION
**Pull request by corredD**
# Description
Enable to switch to illustrative color scheme in the tree. Use previously applied color at entity level so we can change the color. 

Is there an easy way to access more coloring options ?

### CODE REVIEW

**Comment by arose** [comment]
I like the idea of an illustrative option but I think it should be a boolean and would require a custom uniform theme. Allowing all themes has massive performance/memory implications and breaks the color boxes in the UI, your way of enabling the illustrative theme already breaks it.

**Comment by corredD** [comment]
I will look at the broken color boxes, a boolean will not allow for the illustrative per chain theme.

**Comment by corredD** [comment]
What about adding a generative per-chain in the list and a toggle for illustrative ? would that make more sense ?

**Comment by arose** [comment]
That would be great!

**Comment by corredD** [comment]
I made a boolean switch for the illustrative scheme. Doesn't seem to broke anything.

File: src/apps/mesoscale-explorer/ui/entities.tsx [hunk 1]
```diff
@@ -507,8 +507,11 @@ export class GroupNode extends Node<{ filter: string }, { isCollapsed: boolean,
             const c = type === 'generate' ? groupColors[i] : value;
             update.to(entities[i]).update(old => {
                 if (old.type) {
-                    old.colorTheme.params.value = c;
-                    old.colorTheme.params.lightness = lightness;
+                    if (illustrative) {
+                        old.colorTheme = { name: 'illustrative', params: { style: { name: 'uniform', params: { value: c, lightness: lightness } } } };
```
Comment (arose) [inline src/apps/mesoscale-explorer/ui/entities.tsx:511]: `lightness: lightness` => `lightness`

**Comment by arose** [comment]
> I made a boolean switch for the illustrative scheme. Doesn't seem to broke anything.

Nice!

When coloring type is custom in a group node, the illustrative option should be hidden as it does nothing

**Comment by arose** [comment]
nice, thanks

**Pull request merged**